craco.config.cjs

```diff
@@ -100,10 +125,35 @@ module.exports = {
           return true
         })
 
-      // We're currently on Webpack 4.x which doesn't support the `exports` field in package.json.
-      // Instead, we need to manually map the import path to the correct exports path (eg dist or build folder).
-      // See https://github.com/webpack/webpack/issues/9509.
-      webpackConfig.resolve.alias['@uniswap/conedison'] = '@uniswap/conedison/dist'
+      webpackConfig.resolve.plugins = webpackConfig.resolve.plugins.map((plugin) => {
+        // Allow vanilla-extract in production builds.
+        // See https://sandroroth.com/blog/vanilla-extract-cra#production-build.
+        if (plugin instanceof ModuleScopePlugin) {
+          plugin.allowedPaths.push(path.join(__dirname, 'node_modules/@vanilla-extract/webpack-plugin'))
+        }
+
+        return plugin
+      })
+
+      const rules = webpackConfig.module.rules[1].oneOf
+      webpackConfig.module.rules[1].oneOf = rules.map((rule) => {
+        // The fallback rule (eg for dependencies).
+        if (rule.loader && rule.loader.match(/babel-loader/) && !rule.include) {
+          // Allow not-fully-specified modules so that legacy packages are still able to build.
+          rule.resolve = { fullySpecified: false }
+
+          // The class properties transform is required for @uniswap/analytics to build.
+          rule.options.plugins.push('@babel/plugin-proposal-class-properties')
+        }
+        return rule
+      })
+
+      // react-markdown requires path, and Webpack 5 does not polyfill, so we polyfill it.
+      webpackConfig.resolve.fallback = { path: require.resolve('path-browserify') }
```

Comment:
No, because they are accessed two different ways. `ProvidePlugin` adds `process` to the global scope, but `path` is not on the global scope - it is still imported as a package.